### DISCUSSION TITLE
Github Copilot chat: Error: ENOENT: no such file or directory, unlink

### DISCUSSION THREAD

**Discussion by sarimarton** [community/community#56436] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi,

I have GitHub Copilot subscription and Copilot X allowance. I'm on VSCode Insiders and have GitHub Copilot and Copilot Chat installed.

When I issue some /vscode related command in the chat, it gives this error message:

![image](https://github.com/community/community/assets/2169043/55aff0cf-d60b-4587-9cf1-a0ad303b176f)

```
/vscode break out functions from this file into separate files

GitHub Copilot
Error: ENOENT: no such file or directory, unlink '/home/sarim/.vscode-server-insiders/data/User/globalStorage/github.copilot-chat/commandEmbeddings.json'
```

That file indeed doesn't exist, the directory is empty:

![image](https://github.com/community/community/assets/2169043/15d3b102-7737-4964-9c0f-89444c236779)

Any advice?

**Comment by Akash1134** [comment, marked as answer]
Hi @sarimarton ,

Could you please try running VSCode Insiders as an Administrator and let us know if that helps? Sharing the solution from [another discussion](https://github.com/orgs/community/discussions/56155).

Thanks!

**Reply by sarimarton** [reply]
It works now, and the files are now there:

globalStorage/github.copilot-chat/commandEmbeddings.json
globalStorage/github.copilot-chat/settingEmbeddings.json

I have no idea what caused this, I've been using the editor since I reported the issue...
